ci/do_ci.sh

```diff
@@ -205,9 +205,9 @@ elif [[ "$CI_TARGET" == "bazel.compile_time_options" ]]; then
   echo "bazel with different compiletime options build with tests..."
   # Building all the dependencies from scratch to link them against libc++.
   echo "Building..."
-  bazel build ${BAZEL_BUILD_OPTIONS} ${COMPILE_TIME_OPTIONS} -c dbg //source/exe:envoy-static
+  bazel build ${BAZEL_BUILD_OPTIONS} ${COMPILE_TIME_OPTIONS} -c dbg //source/exe:envoy-static --build_tag_filters=-nofips
```

Comment:
Sorry what is the TLDR on all the nofips changes?

Reply:
QUIC relies on some boringssl hooks that aren't in the (current) fips boringssl release.
I suggested to Dan that we just opt QUIC out of the FIPS build because of that.   I didn't realize that our debug build used the fips boringssl build though :-/

Reply:
I think this is the "compile time options" build. 

What is preventing us from just building/testing QUIC in the normal build at this point?

Reply:
I just documented the issue in https://github.com/envoyproxy/envoy/issues/7433, but basically is what Alyssa explained. 

nofip tag is only excluded from compile-time-options build. And in normal build we can still run and test QUIC integration code.

Reply:
OK. Are we building QUIC in the normal release build now though? Or is it only compile time options?

Reply:
Yes, we build and test QUIC in all CI's except for compile-time-options.

Reply:
OK sounds good.